src/components/CustomInput/CustomInput.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from './CustomInput.module.css';
+
+export const CustomInput = ({ ...props }) => {
```

Comment:
why are you spreading props into a new object then using it, you should extract the individual props like this ```{ label }```